### ISSUE TITLE
no-empty-label issue

### ISSUE THREAD

**Issue by wavded** [eslint/eslint#1161]
The `no-empty-label` rule doesn't seem to acknowledge for-in loops:

``` js
label: // good
  while (true) {

  }

label: // good
  for (;;) {

  }

label: // bad - Unexpected label label
  for (var i in {}) {

  }
```
